src/test/java/org/openrewrite/java/testing/junit6/MinimumJreConditionsTest.java

```diff
@@ -0,0 +1,843 @@
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  class MyTest {
+                      @Test
+                      @EnabledOnJre(versions = { 11, 17 })
+                      void testOnJavaSomeJaveVersions() {
+                          System.out.println("Java 17");
+                      }
+                  }
+                  """,
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  class MyTest {
+                      @Test
+                      void testOnJavaSomeJaveVersions() {
+                          System.out.println("Java 17");
+                      }
+                  }
+                  """
+              )
+            );
+        }
+
+        @ParameterizedTest
+        @ValueSource(strings = {
+          "JAVA_21",
+          "JRE.JAVA_21",
+          "value = JRE.JAVA_21",
+          "versions = 21",
+          "versions = { 21 }"
+        })
+        void keepTestEnabledOnNewerJre(String jre) {
+            rewriteRun(
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+                      @Test
+                      @EnabledOnJre(%s)
+                      void testOnlyOnJava21() {
+                          System.out.println("Java 21");
+                      }
+                  }
+                  """.formatted(jre)
+              )
+            );
+        }
+
+        @Test
+        void withMinimumJava21() {
+            rewriteRun(
+              spec -> spec.recipe(new MinimumJreConditions("21")),
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  class MyTest {
+                      @Test
+                      @EnabledOnJre(JRE.JAVA_17)
+                      void testOnJava17() {
+                          System.out.println("Java 17");
+                      }
+
+                      @Test
+                      @EnabledOnJre(JRE.JAVA_21)
+                      void testOnJava21() {
+                          System.out.println("Java 21");
+                      }
+                  }
+                  """,
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  class MyTest {
+
+                      @Test
```

Comment:
By extension, then here too:
```suggestion
                      @Test
                      @EnabledOnJre(JRE.JAVA_21)
```